server/modlog.ts

```diff
@@ -467,6 +469,7 @@ export class Modlog {
 // even though it's a type not a function...
 type ModlogResult = ModlogEntry | undefined;
 
+export const mainModlog = new Modlog(MODLOG_PATH, MODLOG_DB_PATH);
```

Comment:
Does this API need to be changed?

Reply:
Oh, I see, it's used by Modlog.

Reply:
I would call it `modlog` lowercase `m` unless there's a reason otherwise?

Reply:
I guess it conflicts with other variables called `modlog` in this file? This is fine.